cmd/Gopkg.toml

```diff
@@ -43,3 +43,7 @@
 [[constraint]]
   version = "kubernetes-1.10.1"
   name = "k8s.io/client-go"
+
+[[constraint]]
+  branch = "master"
```

Comment:
Now we have to figure out versioning scheme for dependent libs so that we can `pin` to the versions specifically.